### PR TITLE
daemon: rework util_* files

### PR DESCRIPTION
**Pull request by runcom**
rework util_\* files
ping @LK4D4 

this also fixes a typo, sorry for that

Signed-off-by: Antonio Murdaca runcom@linux.com

### CODE REVIEW

**Comment by cpuguy83** [comment]
I don't think it requires a tag.
Also the utils_nounix should be updated... really it's all a little weird.

**Comment by runcom** [comment]
@cpuguy83 yes there's some inconsistency between the files also

**Comment by runcom** [comment]
reworked those files, see if it's working correctly at least

**Comment by jhowardmsft** [comment]
LGTM following IRC conv.

**Comment by runcom** [comment]
ok brainstormed with @jhowardmsft I'm push forcing in a moment

File: daemon/utils_linux.go [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// +build linux
```
Comment (runcom) [inline daemon/utils_linux.go]: no need for this since _linux is understood by go build

File: daemon/utils_unsupported.go [hunk 2]
```diff
@@ -2,12 +2,5 @@
 
 package daemon
 
-func selinuxSetDisabled() {
```
Comment (runcom) [inline daemon/utils_unsupported.go]: these aren't called from non-unix files so no need to be here

**Comment by runcom** [comment]
This isn't correct apart from renaming

**Pull request closed without merging**